docs/ab-tests/browser.md

```diff
@@ -345,12 +345,14 @@ is stored in a cookie for up to 365 days.
 
 ### Can I use redirects in A/B tests to test entirely different pages or layouts?
 
-Yes, you can. We recommend doing this by running an experiment on your [server](https://developer.piwik.org/guides/ab-tests/server#redirects).
-Server side redirects have the advantage that they are more SEO friendly and faster to load for your users.
+Yes, please see below the different ways this can be achieved:
+
+1. When you create your experiment in the UI, under the section “Redirects”, for each variation (including the “Original” variation) you can can enter the Page URL to redirect to and test. ([learn more in this FAQ](https://matomo.org/faq/ab-testing/faq_22493/)). This is the easiest way to compare the performance of different page URLs.
```

Comment:
Do we need to mention through the UI will result in a JS redirect which is not recommended?